pkgs/development/python-modules/funcsigs/default.nix

```
@@ -11,11 +11,14 @@ buildPythonPackage rec {
     sha256 = "0l4g5818ffyfmfs1a924811azhjj8ax9xd1cffr1mzd3ycn0zfx7";
   };
 
-  buildInputs = [ unittest2 ];
-
   # https://github.com/testing-cabal/funcsigs/issues/10
   patches = lib.optional (isPyPy && isPy3k) [ ./fix-pypy3-tests.patch ];
 
+  # requires, unittest2 and package hasn't been maintained since 2013
+  doCheck = false;
```

```suggestion
  # requires unittest2 and package hasn't been maintained since 2013
  doCheck = false;
```
but why not apply your upstream PR using fetchpatch?

---

I didn't make a PR for funcsigs, just `deprecation`

---

deprecation has had recent activity upstream. funcsigs is pretty much unmaintained as it's just a backport package. I'm assuming that people still listing this explicitly could be using stdlib.